modules/init.go

```diff
@@ -3,80 +3,81 @@
-	_ "github.com/netdata/go.d.plugin/modules/freeradius"
-	_ "github.com/netdata/go.d.plugin/modules/geth"
-	_ "github.com/netdata/go.d.plugin/modules/haproxy"
-	_ "github.com/netdata/go.d.plugin/modules/hdfs"
-	_ "github.com/netdata/go.d.plugin/modules/httpcheck"
-	_ "github.com/netdata/go.d.plugin/modules/isc_dhcpd"
-	_ "github.com/netdata/go.d.plugin/modules/k8s_kubelet"
-	_ "github.com/netdata/go.d.plugin/modules/k8s_kubeproxy"
-	_ "github.com/netdata/go.d.plugin/modules/k8s_state"
-	_ "github.com/netdata/go.d.plugin/modules/lighttpd"
-	_ "github.com/netdata/go.d.plugin/modules/lighttpd2"
-	_ "github.com/netdata/go.d.plugin/modules/logind"
-	_ "github.com/netdata/go.d.plugin/modules/logstash"
-	_ "github.com/netdata/go.d.plugin/modules/mongodb"
-	_ "github.com/netdata/go.d.plugin/modules/mysql"
-	_ "github.com/netdata/go.d.plugin/modules/nginx"
-	_ "github.com/netdata/go.d.plugin/modules/nginxvts"
-	_ "github.com/netdata/go.d.plugin/modules/nvidia_smi"
-	_ "github.com/netdata/go.d.plugin/modules/openvpn"
-	_ "github.com/netdata/go.d.plugin/modules/openvpn_status_log"
-	_ "github.com/netdata/go.d.plugin/modules/pgbouncer"
-	_ "github.com/netdata/go.d.plugin/modules/phpdaemon"
-	_ "github.com/netdata/go.d.plugin/modules/phpfpm"
-	_ "github.com/netdata/go.d.plugin/modules/pihole"
-	_ "github.com/netdata/go.d.plugin/modules/pika"
-	_ "github.com/netdata/go.d.plugin/modules/ping"
-	_ "github.com/netdata/go.d.plugin/modules/portcheck"
-	_ "github.com/netdata/go.d.plugin/modules/postgres"
-	_ "github.com/netdata/go.d.plugin/modules/powerdns"
-	_ "github.com/netdata/go.d.plugin/modules/powerdns_recursor"
-	_ "github.com/netdata/go.d.plugin/modules/prometheus"
-	_ "github.com/netdata/go.d.plugin/modules/proxysql"
-	_ "github.com/netdata/go.d.plugin/modules/pulsar"
-	_ "github.com/netdata/go.d.plugin/modules/rabbitmq"
-	_ "github.com/netdata/go.d.plugin/modules/redis"
-	_ "github.com/netdata/go.d.plugin/modules/scaleio"
-	_ "github.com/netdata/go.d.plugin/modules/snmp"
-	_ "github.com/netdata/go.d.plugin/modules/solr"
-	_ "github.com/netdata/go.d.plugin/modules/springboot2"
-	_ "github.com/netdata/go.d.plugin/modules/squidlog"
-	_ "github.com/netdata/go.d.plugin/modules/supervisord"
-	_ "github.com/netdata/go.d.plugin/modules/systemdunits"
-	_ "github.com/netdata/go.d.plugin/modules/tengine"
-	_ "github.com/netdata/go.d.plugin/modules/traefik"
-	_ "github.com/netdata/go.d.plugin/modules/unbound"
-	_ "github.com/netdata/go.d.plugin/modules/vcsa"
-	_ "github.com/netdata/go.d.plugin/modules/vernemq"
-	_ "github.com/netdata/go.d.plugin/modules/vsphere"
-	_ "github.com/netdata/go.d.plugin/modules/weblog"
-	_ "github.com/netdata/go.d.plugin/modules/whoisquery"
-	_ "github.com/netdata/go.d.plugin/modules/wireguard"
-	_ "github.com/netdata/go.d.plugin/modules/wmi"
-	_ "github.com/netdata/go.d.plugin/modules/x509check"
-	_ "github.com/netdata/go.d.plugin/modules/zookeeper"
+	//_ "github.com/netdata/go.d.plugin/modules/activemq"
```

Comment:
Why are these all commented out?

Reply:
yeah, forgot to undo 🙈 thanks!

Reply:
fixed in https://github.com/netdata/go.d.plugin/pull/975